<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVPStudio MXLPlayer - Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutMXLLoad">
      <item>
       <widget class="QLabel" name="labelMXL">
        <property name="text">
         <string>MXL文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMXLPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMXLBrowse">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutWAVLoad">
      <item>
       <widget class="QLabel" name="labelWAV">
        <property name="text">
         <string>WAV文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditWAVPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonWAVBrowse">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>尺寸选择</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonSmall">
         <property name="text">
          <string>Small - 5.5M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonMedium">
         <property name="text">
          <string>Medium - 9M</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonLarge">
         <property name="text">
          <string>Large - 12M</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="pushButtonPlay">
        <property name="text">
         <string>播放...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExportMP4">
        <property name="text">
         <string>导出MP4...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
